tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs

```diff
@@ -1590,6 +1591,37 @@ private List<SelectListItem> CreateTestTeamsList()
             };
         }
 
+        private TournamentTeamsListViewModel CreateExpectedTeamsList()
+        {
+            TournamentTeamsListViewModel expectedTeams = new TournamentTeamsListViewModel
+            {
+                TeamsList = new List<TeamNameViewModel>()
+            {
+                new TeamNameViewModel()
+                {
+                    DivisionName="DivisionNameA",
+                    Id=1,
+                    Name="TeamNameA"
+                },
+                new TeamNameViewModel()
+                {
+                    DivisionName="DivisionNameA",
+                    Id=2,
+                    Name="TeamNameB"
+                },
+                new TeamNameViewModel()
+                {
+                    DivisionName="DivisionNameA",
+                    Id=3,
+                    Name="TeamNameC"
+                }
+            },
+                TournamentId = TEST_DIVISION_ID
+            };
+
+            return expectedTeams;
+        }
+
```

Comment:
Please format code.

Reply:
I don't know exactly how should it look like?

Reply:
http://visualstudioshortcuts.com/2017/
Search for 'format document'

Reply:
What exactly should change?

Reply:
Did you try "Ctrl+K, Ctrl+D" keyboard shortcut in this file?

Reply:
Yes, here is a new version.

Reply:
@MikolaB on his laptop Format Document command is not working 😮